docs/faq.md

```diff
@@ -97,6 +98,38 @@ Next automatically defines `React` for all of your files via a babel plugin. In
 1.  Adding `import React from 'react'` to your component files.
 2.  Adding a `.babelrc` that includes [`babel-plugin-react-require`](https://www.npmjs.com/package/babel-plugin-react-require)
 
+### How do I fix module resolution while using pnpm Plug-n-Play?
+
+In case you are using [pnpm](https://pnpm.io/) you might run into issues with module resolution similar to this:
+
+```text
+WARN   Failed to load preset: "@storybook/react-webpack5/preset"`
+Required package: @storybook/react-webpack5 (via "@storybook/react-webpack5/preset")
+```
+
+To fix this, you can wrap the package name like this:
+
+```ts
+const path = require('path');
+
```

Comment:
@anarmawala, when you're able, can you make a small adjustment to the snippet mentioning the file that will be updated, in this case, `.storybook/main.ts` also, I believe you can replace the `require` with an actual import and will work. Which turns this snippet into:

```ts
// .storybook/main.ts

import path from 'path';

// the rest of the code for the snippet
```